exporters/trace/jaeger/uploader.go

```diff
@@ -38,24 +38,40 @@ type batchUploader interface {
 type EndpointOption func() (batchUploader, error)
 
 // WithAgentEndpoint instructs exporter to send spans to jaeger-agent at this address.
-// For example, localhost:6831.
-func WithAgentEndpoint(agentEndpoint string, options ...AgentEndpointOption) EndpointOption {
+// Agent endpoint is resolved in the following order: options, env vars, and then default values
+// Default agent endpoint value: localhost:6832
```

Comment:
```suggestion
// WithAgentEndpoint configures the Jaeger exporter to send spans to a jaeger-agent. This will
// use the following environment variables for configuration if no explicit option is provided:
//
// - OTEL_EXPORTER_JAEGER_AGENT_HOST is used for the agent address host
// - OTEL_EXPORTER_JAEGER_AGENT_PORT is used for the agent address port
// 
// The passed options will take precedence over any environment variables and default values
// will be used if neither are provided.
```